src/views/PatronsPreRegistrationListContainer/PatronsPreRegistrationList.js

```diff
@@ -0,0 +1,75 @@
+import PropTypes from 'prop-types';
+import { MultiColumnList, Button } from '@folio/stripes/components';
+import { useIntl } from 'react-intl';
+import { get, noop } from 'lodash';
+
+import { visibleColumns, columnMapping, COLUMNS_NAME } from './constants';
+
+const PatronsPreRegistrationList = ({
+  data,
+  isEmptyMessage,
+  totalCount,
+  onNeedMoreData
+}) => {
+  const intl = useIntl();
+
+  const preRegistrationsListFormatter = () => ({
+    [COLUMNS_NAME.ACTION]: () => (
+      <Button
+        type="button"
+        onClick={noop} // this will be updated in the scop of next ticket
+      >
+        New
```

Comment:
`<FormattedMessage id="stripes-components.addNew" />` will give you a `New` label. `stripes-core.button.new` gives you `+ New`.

Reply:
Hello @zburke ,
Users interface provides this end point, the BE version will be increased and merged to master soon.
CC: @Terala-Priyanka

Reply:
Then you have two options:
1. add `"staging-users": "1.0"` to `package.json` under `okapiInterfaces` and bump the major version of this module.
2. add `"staging-users": "1.0"` to `package.json` under `optionalOkapiInterfaces` and fence off all code that uses it behind `<IfInterface>` conditionals, leaving the version of this module as-is.

Reply:
@zburke 
we will add "staging-users": "1.0" to package.json under okapiInterfaces (once it is available), mark as breaking in change log, but will retain the major version as is, because it has been bumped up already in the scope of Ramsons.

Reply:
Additionally, the description for [UIU-3219](https://folio-org.atlassian.net/browse/UIU-3219) states "As a LoC registration staff..." Stripes cannot make this assertion; it has no notion of a specific tenant. This ticket, and therefore this PR, must not be merged as-is because they make an assertion that cannot be evaluated.

The closest we can come would be to wrap this feature behind a new permission set, but this implementation uses an existing one, `ui-users.create`.

Reply:
we were trying to achieve some parallellism in the interest of time.

Reply:
Hello @zburke , 
Can you please review this PR on urgent basis we are running against time atm :)
Thanks for your immediate attention.

Reply:
Sorry, @gurleenkaurbp, but I cannot approve this as is. I disagree with how the story was written and I disagree with how this PR implements it. This change will show the patron pre-registration list to users with "create" permission; that is wrong. I understand that you attempted to break up this story by putting the new permission set in a separate story; that is wrong. 

To do this work in a way that avoids impacting existing implementation, this new feature must be implemented behind a conditional such as a feature flag, an optional interface dependency, or a permission set. None of those conditionals are implemented here. I understand the permission set will be added in UIU-3221. Fine, then block this ticket on that one and implement that story first. 

I'm sorry if this seems petty but IMO this work is not being done safely or correctly. I will dismiss my "request-changes" review but I will not approve this as is.

Reply:
Hello @zburke 
Please take a look at the pr's I hope they suffice now for the approval